### ISSUE TITLE
Cannot read property 'toLowerCase' of undefined

### ISSUE THREAD

**Issue by alehaa** [ToddGreenfield/homebridge-onkyo#89]
**Confirm you have checked the WIKI for troubleshooting first**
_Fill out and check (`[x]`) the boxes which apply.
- Homebridge version: v1.2.4
- Plugin version: 0.8.1
- OS, including release name/version: alpine (Docker)
- [ ] Homebridge-Onkyo installed via OS/distribution mechanisms
- [ ] You installed using homebridge-config-ui-x
- [ ] You enabled homebridge debug output

**Describe the bug**
The change introduced in b2f6d4462b91ff7493b99a709db09bcc91a59a8b converts all zone names to lowercase. However, if no zone is defined, `toLowerCase()` will be called on an undefined object, which in turn crashes homebridge. Therefore, the default value needs to be applied *before* the method is called.

**To Reproduce**
Information from the log:
```
[12/19/2020, 6:36:08 PM] [Onkyo] Initializing Onkyo platform...
[12/19/2020, 6:36:08 PM] [Onkyo] **************************************************************
[12/19/2020, 6:36:08 PM] [Onkyo]   homebridge-onkyo version 0.8.1
[12/19/2020, 6:36:08 PM] [Onkyo]   GitHub: https://github.com/ToddGreenfield/homebridge-onkyo 
[12/19/2020, 6:36:08 PM] [Onkyo] **************************************************************
[12/19/2020, 6:36:08 PM] [Onkyo] start success...
[12/19/2020, 6:36:08 PM] TypeError: Cannot read property 'toLowerCase' of undefined
    at new OnkyoAccessory (/usr/local/lib/node_modules/homebridge-onkyo/index.js:70:32)
    at /usr/local/lib/node_modules/homebridge-onkyo/index.js:37:22
    at Array.forEach (<anonymous>)
    at OnkyoPlatform.createAccessories (/usr/local/lib/node_modules/homebridge-onkyo/index.js:30:13)
    at new OnkyoPlatform (/usr/local/lib/node_modules/homebridge-onkyo/index.js:23:8)
    at /usr/local/lib/node_modules/homebridge/src/server.ts:395:40
    at Array.forEach (<anonymous>)
    at Server.loadPlatforms (/usr/local/lib/node_modules/homebridge/src/server.ts:372:27)
    at Server.start (/usr/local/lib/node_modules/homebridge/src/server.ts:151:29)
    at cli (/usr/local/lib/node_modules/homebridge/src/cli.ts:80:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
[12/19/2020, 6:36:08 PM] Got SIGTERM, shutting down Homebridge...
```

**Comment by jondthompson** [comment]
It took me longer to figure out how to cherry pick the one commit than it did to solve the bug :D

**Comment by cbrandlehner** [comment]
published release 0.8.2 to address this issue.

**Comment by alehaa** [comment]
Thanks for the quick response! :)
